src/workerfarm/WorkerFarm.js

```diff
@@ -275,8 +264,16 @@ class WorkerFarm extends EventEmitter {
     return cores || 1;
   }
 
+  static callMaster(request, awaitResponse = true) {
+    if (process.parcelWorker) {
+      return process.parcelRequest(request, awaitResponse);
```

Comment:
`child` is a singleton, so you could probably avoid attaching stuff to `process` by just requiring it here:

```javascript
if (process.send) {
  let child = require('./child');
  return  child.addCall(request, awaitResponse);
}
// ...
```

Reply:
Just checking process.send is pretty dangerous as parcel can be used as an api in a worker sandbox in node.js

Reply:
mmm, maybe something like `require.main === childModule`? That would check if the module the process launched with is the parcel child, which is basically what we want.